### ISSUE TITLE
[2.0.0] getLocalIdent validation

### ISSUE THREAD

**Issue by gpoitch** [webpack-contrib/css-loader#862]
* Operating System: macOS 10.14.1
* Node Version: 11.2.0
* NPM Version: 6.4.1
* webpack Version: 4.27.1
* css-loader Version: 2.0.0

### Expected Behavior

Previously, if `getLocalIdent` returned a falsey value, css-loader would fallback to using `localIdentName`

### Actual Behavior

css-loader throws an error:
```
ValidationError: CSS Loader Invalid Options
options.getLocalIdent should pass "instanceof" keyword validation
```

### Code

```js
const isProduction = false

const cssLoader = {
  loader: 'css-loader',
  options: {
    modules: true,
    localIdentName: '[local]__[hash:base64:3]',
    getLocalIdent: isProduction && function() {
       return 'foo' // other logic here, but just return a string for purpose of issue
    }
  }
}
```

### How Do We Reproduce?

Run tests with those loader options

**Comment by evilebottnawi** [comment]
@gpoitch what you expected from `getLocalIdent` when you set it `false`?

**Comment by gpoitch** [comment]
I expect it to fallback to using `localIdentName`.
Worked around it by explicitly setting to `undefined` instead of a falsey value.
```js
getLocalIdent: isProduction ? function(){ /*...*/ } : undefined
```

**Comment by evilebottnawi** [comment]
@gpoitch maybe we can accept `false` value for easy configuration, what do you think?

**Comment by gpoitch** [comment]
@evilebottnawi yes, that'd be nice, as it worked in 1.x.  Otherwise that is a tiny breaking change

**Comment by evilebottnawi** [comment]
@gpoitch let's mark it as bug and fix in `2.0.1`

**Comment by evilebottnawi** [comment]
previous we don't have schema options and tests so i can lose something, sorry
